### PR TITLE
Introduced image pull policy support (#832)

### PR DESCRIPTION
**Pull request by akranga**
Added image_pull_policy attribute to `ContainerOp`

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/942)

### CODE REVIEW

File: sdk/python/kfp/dsl/_container_op.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def __init__(self, name: str, image: str, command: str=None, arguments: str=None
     self.human_name = name
     self.name = _pipeline.Pipeline.get_default_pipeline().add_op(self, is_exit_handler)
     self.image = image
+    self.image_pull_policy = None
```
Comment (Ark-kun) [inline sdk/python/kfp/dsl/_container_op.py]: Can we make the field private (leading underscore) and add a setter function? `set_image_pull_policy `

**Comment by mattiasarro** [comment]
@Ark-kun how can we make progress on this PR? Should I create a new branch, apply the changes you required (private variable + setter method), and create a new PR? Or @akranga is planning to implement the changes any time soon?

**Comment by vicaire** [comment]
@akranga, can we close this PR given @mattiasarro comment?

**Comment by akranga** [comment]
This is great! Let's close PR as kwargs provide much more scalable implementation

**Comment by vicaire** [comment]
Closing. Thanks!

**Pull request closed without merging**